src/observable/FromEventObservable.ts

```diff
@@ -133,6 +133,8 @@ export class FromEventObservable<T, R> extends Observable<T> {
       const source = sourceObj;
       sourceObj.addListener(eventName, handler);
       unsubscribe = () => source.removeListener(eventName, handler);
+    } else {
+      throw new TypeError('Invalid event target');
```

Comment:
I feel like we're not capitalizing our error messages anywhere else. Can you confirm? cc/ @kwonoj